src/stashbox.extension.wcf/PerServiceInstanceLifetime.cs

```diff
@@ -8,14 +8,12 @@ namespace Stashbox.Extension.Wcf
 {
     public sealed class PerServiceInstanceLifetime : LifetimeBase
     {
+        private volatile Expression _expression;
```

Comment:
@z4kn4fein Hi Peter, I am somewhat familiar with the functionality of the "volatile" keyword and mostly just curious than anything of its use-case here?